lib/rubocop/formatter/formatter_set.rb

```diff
@@ -73,7 +76,7 @@ def add_formatter(formatter_type, output_path = nil)
           output = $stdout
         end
 
-        self << formatter_class.new(output)
+        self << formatter_class.new(output, @options)
```

Comment:
Btw, I'm not sure how many custom formatters exist in the wild, so changing this is not out of question. And we're not at 1.0 yet. :-)